packages/cli/src/gemini.tsx

```diff
@@ -158,32 +158,19 @@ export async function startInteractiveUI(
   resumedSessionData: ResumedSessionData | undefined,
   initializationResult: InitializationResult,
 ) {
-  // When not in screen reader mode, disable line wrapping.
-  // We rely on Ink to manage all line wrapping by forcing all content to be
-  // narrower than the terminal width so there is no need for the terminal to
-  // also attempt line wrapping.
-  // Disabling line wrapping reduces Ink rendering artifacts particularly when
-  // the terminal is resized on terminals that full respect this escape code
-  // such as Ghostty. Some terminals such as Iterm2 only respect line wrapping
-  // when using the alternate buffer, which Gemini CLI does not use because we
-  // do not yet have support for scrolling in that mode.
-  if (!config.getScreenReader()) {
```

Comment:
this functionality was moved into ink as the incremental rendering algorithm requires it for robustness.